### ISSUE TITLE
[Sidebar] - Sidebar transition using wrong transtion

### ISSUE THREAD

**Issue by tmyler** [Semantic-Org/Semantic-UI#6514]
**Steps to Reproduce**
1. Click back and forth between the two buttons on the right of the page, several times at a quick rate.
2.  Stop clicking, see that the buttons get pushed across an extra amount, the yellow main segment div becomes visible as the grey pusher div is pushed across. 

The pusher is being moved as if it has the "push" transition and not the "overlay" transition that is set in the JS. 

**Expected**
The Sidebar should always use the overlay transition

**Result**
The Sidebar does not always use the overlay transition

**Testcase**
http://jsfiddle.net/tmyler/rduvhn8u/334/

**Comment by hammy2899** [comment]
I can't seem to reproduce this issue, what browser are you using?

**Comment by tmyler** [comment]
It occurs in both:
Firefox Quantum 61.0.1 (64 bit) 
Chrome Version 68.0.3440.84 (Official Build) (64-bit)
My PC is running Windows 10 Pro Version 1809 Build 17134.165

Here is quick video of the bug occurring for me in Firefox. 
[Edit fiddle - JSFiddle - Mozilla Firefox 03_08_2018 08_50_07.zip](https://github.com/Semantic-Org/Semantic-UI/files/2256510/Edit.fiddle.-.JSFiddle.-.Mozilla.Firefox.03_08_2018.08_50_07.zip)

**Comment by hammy2899** [comment]
Ok I see now, that is a weird one. It's as if the transitions are queued and because the buttons are spammed it can't keep up therefore it leads to running the transitions incorrectly.

**Comment by tmyler** [comment]
Yes it is definitely some sort of strange issue with queued inputs. It may be something that cannot be fixed but is an interesting bug either way.

**Comment by tmyler** [comment]
Has there been any developments in investigating or solving this issue?

**Comment by hammy2899** [comment]
@tmyler you may or may not know that SUI development has slowed down recently so this issue will probably not be resolved for quite some time.
